shared/insideResinOS/2.0.md

```diff
@@ -0,0 +1,154 @@
+
+### Standalone ResinOS
+
+ResinOS also comes in a Standalone or "Unmanaged" variant. This variant is exactly the same as the `-dev` variant but **does not** have the resin.io supervisor agent and has the resin VPN service disabled, so it will never connect to the resin.io management system.
+
+The standalone version of resinOS is meant as an excellent way to get started with docker containers on embedded systems and you can read more about this over at [resinos.io](https://resinos.io/).
+
+## ResinOS Components
+The resinOS userspace tries to package only the bare essentials for running containers while still offering a lot of flexibility. The philosophy is that software and services always default to being in a container, unless they are generically useful to all containers or they absolutely can’t live in a container.  The userspace consists of many open source components, but in this section we will just highlight some of the most important services.
+
+<img src="/img/common/resinos/resinOS-components.png" width="90%">
+
+### Resin.io Supervisor
+
+The resin.io supervisor is a lightweight container which runs on your device, manages your applications and communicates with our servers - downloading new application containers and updates to existing containers as you push them, sending logs to your dashboard. It also provides a useful [HTTP supervisor API interface](https://docs.resin.io/runtime/supervisor-api/), which allows you to query update status and perform certain actions on the device.
+
+### Systemd
+We use systemd as the init system for resinOS and it is responsible for launching and managing all the other services. We leverage many of the great features of systemd, such as adjusting OOM scores for critical services and running services in separate mount namespaces. Systemd also allows us to easily manage service dependencies.
+
+### Docker
+The docker engine is a lightweight container runtime that allows us to build and run linux containers on resinOS. ResinOS has been optimized to run docker containers and has been set up to use the journald log driver and DNSmasq for container DNS resolution.
+We use AUFS as the underlying storage driver since it is arguably the most production tested storage driver in the docker ecosystem. It also allows us to more easily support devices with older kernel versions and additionally gives us the ability to run on devices with Unmanaged NAND flash.
+
+### NetworkManager and ModemManager
+ResinOS uses NetworkManager accompanied by ModemManager, to deliver a stable and reliable connection to the internet, be it via ethenet, wifi or cellular modem. Additionally to make headless configuration of the device’s network easy, we have added a `system-connections` folder in the boot partition which is copied into `/etc/NetworkManager/system-connections`. So any valid NetworkManager connection file can just be dropped into the boot partition before device commissioning.
```

Comment:
should we clarify, where appropriate, that this is resinOS 2.0 that we are talking about? Since some people will still be running 1.x where this isn't true?